### DISCUSSION TITLE
[Snowflake] Is skip "white" caracter needed ?

### DISCUSSION THREAD

**Discussion by MasterKuat** [antlr/grammars-v4#3105] [Q&A] [answered]
The lexer line below make output useless to me
https://github.com/antlr/grammars-v4/blob/3128642ce4e274084439d7a4e4d329826a3e4ea3/sql/snowflake/SnowflakeLexer.g4#L1065

Should be better to set to "channel(HIDDEN)", should not ?

**Comment by mlorek** [comment, marked as answer]
it should.
fixed in #3100
